bug-bounty-hof/web.yml

```diff
@@ -68,8 +101,7 @@ names:
   date: 2022-05-16
 - name: Ali Syarief
   date: 2022-05-11
-  twitter: "@kang_ali"
-  url: https://twitter.com/@kang_ali
+  url: https://www.linkedin.com/in/mohammad-ali-syarief/
 - name: Brian Carpenter of Geeknik Labs & Farm
```

Comment:
@tomrittervg this reporter declined to be mentioned in HoF https://bugzilla.mozilla.org/show_bug.cgi?id=1796601#c10

Reply:
He's not in Q4; this entry is for some other bug in Q2..